lib/shared/common_markdown_body.dart

```diff
@@ -43,19 +45,32 @@ class CommonMarkdownBody extends StatelessWidget {
       data: body,
       inlineSyntaxes: [LemmyLinkSyntax()],
       imageBuilder: (uri, title, alt) {
-        return Padding(
-          padding: const EdgeInsets.symmetric(vertical: 8.0),
-          child: Row(
-            mainAxisAlignment: MainAxisAlignment.start,
-            children: [
-              ImagePreview(
-                url: uri.toString(),
-                isExpandable: true,
-                isComment: isComment,
-                showFullHeightImages: true,
+        return FutureBuilder(
+          future: () {
+            return isImageUriSvg(uri);
+          }(),
+          builder: (BuildContext context, AsyncSnapshot<dynamic> snapshot) {
```

Comment:
I would change this to show the correct response type instead of `dynamic`!

```dart
builder: (BuildContext context, AsyncSnapshot<bool> snapshot) {
```

Reply:
Yep, done!